components/tooltip/index.css

```diff
@@ -647,3 +570,14 @@ governing permissions and limitations under the License.
 		}
 	}
 }
+
+@media (forced-colors: active) {
+	.spectrum-Tooltip {
+		border: 1px solid transparent;
+	}
+
+	.spectrum-Tooltip-tip {
+		forced-color-adjust: none;
+		--highcontrast-tooltip-background-color-default: CanvasText;
```

Comment:
This could probably be moved underneath `@media (forced-colors: active)` directly.

Reply:
👍 Gotcha. I had thought the main background and tip background were using different custom properties.